src/assessment/builder.py

```diff
@@ -165,6 +173,11 @@ def build_campaigns(assessment, campaign_number, template_smtp):
         else:
             logging.error("Complete Date is not after Launch Date.")
```

Comment:
It's outside the scope of this PR, but since it caught my eye, here is some unnecessary capitalization that should be fixed:
```suggestion
            logging.error("Complete date is not after launch date.")
```

Reply:
Should be changed here: 5a0cc25762521a630a682bdebea6964e12f4dbe9

Reply:
My suggestion was not included in https://github.com/cisagov/gophish-tools/commit/5a0cc25762521a630a682bdebea6964e12f4dbe9.  Feel free to use the "Commit suggestion" button above.